### PR TITLE
Bigint felt zero tests

### PR DESCRIPTION
**Pull request by martinacantaro**
# Bigint felt zero tests

## Description

I'm reusing old unit tests that have been rendered obsolete by proptests (that test the same operations but better) and converting them to unit tests for the edge case where felts are equal to 0. 

Related issue: https://github.com/lambdaclass/cairo-rs/issues/826

## Checklist
- [x] Linked to Github Issue
- [x] Unit tests added
- [ ] Integration tests added.
- [x] This change requires new documentation.
  - [x] Documentation has been added/updated. Tests are documented in the code comments

### CODE REVIEW

**Review by Juan-M-V — APPROVED** [review]

File: felt/src/bigint_felt.rs [hunk 1]
```diff
@@ -820,60 +820,106 @@ impl fmt::Display for ParseFeltError {
...
+        let c = FeltBigInt::new(0);
 
         assert_eq!(a + b, c);
     }
 
     #[test]
-    fn add_assign_felts_within_field() {
-        let mut a = FeltBigInt::<FIELD_HIGH, FIELD_LOW>::new(1i32);
-        let b = FeltBigInt::new(2i32);
+    // Tests that the result of performing add asign with two zeros is zero.
```
Comment (Jrigada) [inline felt/src/bigint_felt.rs:836]:
```suggestion
    // Tests that the result of performing add assign with two zeros is zero.
```

**Review by Jrigada — APPROVED** [review]
